### PR TITLE
Function arguments

### PR DESCRIPTION
**Pull request by narodejesus**

### CODE REVIEW

**Comment by jdmunro** [comment]
Nice contribution @narodejesus 

A couple of points:

- Headers should be title cased (e.g. Function arguments -> Function Arguments)
- I think it is important we justify the _why_ of the rule. So in this case, there should be a small bit of introductory text that describes why we prefer this style 😄

File: README.md [hunk 1]
```diff
@@ -152,7 +152,33 @@ const add = function (a, b) {
 }
 
 // good
-const add = (a, b) => a + b 
+const add = (a, b) => a + b
+```
+
+### Function arguments
+function parameters should be less than to 4
```
Comment (jdmunro) [inline README.md]: Needs a capital 'f'

File: README.md [hunk 2]
```diff
@@ -152,7 +152,33 @@ const add = function (a, b) {
...
+}
+
+// good (destructuring and use of default parameters)
+const showNotification = ({
+	id,
+	title = 'Default Notification Title',
+	message,
+	duration,
+	type,
+}) => {
```
Comment (DaleJefferson) [inline README.md]: I think we should introduce Flow into the handbook.

Comment (DaleJefferson) [inline README.md]: Lets do this in another PR
